packages/e2e-tests/specs/performance/site-editor.test.js

```diff
@@ -58,13 +87,59 @@ describe( 'Site Editor Performance', () => {
 			results.load.push( new Date() - startTime );
 		}
 
+		// Measuring typing performance inside the post content.
+		await canvas().click(
+			'[data-type="core/post-content"] [data-type="core/paragraph"]'
+		);
+		await insertBlock( 'Paragraph' );
+		i = 200;
+		const traceFile = __dirname + '/trace.json';
+		await page.tracing.start( {
+			path: traceFile,
+			screenshots: false,
+			categories: [ 'devtools.timeline' ],
+		} );
+		while ( i-- ) {
+			await page.keyboard.type( 'x' );
+		}
+		await page.tracing.stop();
+		const traceResults = JSON.parse( readFile( traceFile ) );
+		const [
+			keyDownEvents,
+			keyPressEvents,
+			keyUpEvents,
+		] = getTypingEventDurations( traceResults );
+
+		// Both keydown and keypress events are bubbled from the iframe to the
+		// main frame, which must be ignored. These will be the odd values in
+		// the array.
```

Comment:
We should be able to remove this soon when we remove the event bubbling from the iframe.